pmd-apex/src/main/resources/category/apex/errorprone.xml

```diff
@@ -443,4 +443,77 @@ private class TestClass {
 ]]>
        </example>
     </rule>
+    
+        <rule name="InaccessibleAuraEnabledGetter"
+          language="apex"
+          since="6.36.0"
+          message="AuraEnabled getter must be public or global if is referenced in Lightning components"
+          class="net.sourceforge.pmd.lang.apex.rule.errorprone.InaccessibleAuraEnabledGetterRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_apex_errorprone.html#inaccessibleauraenabledgetter">
+        <description>
+			In the Summer '21 release, a mandatory security update enforces access modifiers on Apex properties in Lightning component markup.
+			The update prevents access to private or protected Apex getters from Aura and Lightning Web Components.
+        </description>
+        <priority>3</priority>
+        <example>
+<![CDATA[// Violating
+public class Foo {
+    @AuraEnabled
+    public Integer counter { private get; set; }
+
+    @AuraEnabled
+    public static Foo bar()
+    {
+        Foo foo = new Foo();
+        foo.counter = 2; 
+        return foo;
+    }
+}
+
+public class Foo {
+    @AuraEnabled
+    public static Baz bar()
+    {
+    	Baz baz = new Baz();
+        baz.counter = 2; 
+        return baz;
+    }
+    
+    private class Baz {
+        @AuraEnabled
+        public Integer counter { protected get; set; }
+    }
+}
+
+// Compliant
+public class Foo {
+    @AuraEnabled
+    public Integer counter { get; set; }
```

Comment:
Please add a comment on this line to explain how this is compliant, like `// a public getter is compliant`